Sources/OktaOidc/Common/OktaOidcError.swift

```diff
@@ -13,7 +13,8 @@
 import Foundation
 
 public enum OktaOidcError: Error {
-    case APIError(String)
+    
+    case api(message: String, underlineError: Error?)
```

Comment:
Actually I think `network` would be better naming, but to be consistent and keep the same naming I left as `api`.